src/hotspot/cpu/x86/macroAssembler_x86.cpp

```diff
@@ -912,7 +912,7 @@ static int reg2offset_out(VMReg r) {
 }
 
 // A long move
-void MacroAssembler::long_move(VMRegPair src, VMRegPair dst) {
+void MacroAssembler::long_move(VMRegPair src, VMRegPair dst, Register tmp, int in_stk_bias, int out_stk_bias) {
```

Comment:
A 'bias' is added to all stack offsets to account for the 32 bytes of shadow space needed for the Window C ABI. Also, the temp register is made configurable.